client/src/Components/Stars/OneStar.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+import styles from './FiveStar.css';
+
+const FourStar = () => {
```

Comment:
Just check for naming for the rest of the stars, looks like they all say FourStar